potodo/_po_file.py

```diff
@@ -16,9 +16,17 @@ def __init__(self, path: Path):
         self.pofile: polib.POFile = polib.pofile(self.path)
         self.directory: str = self.path.parent.name
 
-        self.fuzzy_entries: Sequence[polib.POEntry] = self.pofile.fuzzy_entries()
```

Comment:
What about `= [entry for entry in self.pofile.fuzzy_entries() if not entry.obsolete]` ?
Or even
`= [entry for entry in self.pofile if entry.fuzzy and not entry.obsolete]` ?